spec/features/errors_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "custom error handler page" do
+  describe "with custom handler setup" do
+
+    around :each do |example|
+      Rails.application.config.consider_all_requests_local = false
+      Rails.application.config.action_dispatch.show_exceptions = true
+      Rails.env = ActiveSupport::StringInquirer.new("production")
+      example.run
+      Rails.application.config.consider_all_requests_local = true
+      Rails.application.config.action_dispatch.show_exceptions = false
+      Rails.env = ActiveSupport::StringInquirer.new("test")
+    end
```

Comment:
I think it would be safer to rewrite the code in that way:

```ruby
around :each do |example|
  begin
    Rails.application.config.consider_all_requests_local = false
    Rails.application.config.action_dispatch.show_exceptions = true
    Rails.env = ActiveSupport::StringInquirer.new("production")
    example.run
  ensure
    Rails.application.config.consider_all_requests_local = true
    Rails.application.config.action_dispatch.show_exceptions = false
    Rails.env = ActiveSupport::StringInquirer.new("test")
  end
end
```